external/test-runtime/XUnit.Runtime.depproj

```diff
@@ -19,7 +19,7 @@
     <XUnitAdapter Condition="'$(TargetGroup)' == 'netcoreapp'">xunit.runner.visualstudio.dotnetcore.testadapter</XUnitAdapter>
     <XUnitAdapter Condition="'$(TargetGroup)' == 'uap' OR '$(TargetGroup)' == 'uapaot'">xunit.runner.visualstudio.uwp.testadapter</XUnitAdapter>
     <XUnitAdapter Condition="'$(TargetGroup)' == 'netfx'">xunit.runner.visualstudio.testadapter</XUnitAdapter>
-    <MicrosoftNetTestSdkPackageName>Microsoft.NET.Test.Sdk</MicrosoftNetTestSdkPackageName>
+    <MicrosoftNetTestSdkPackageName>microsoft.net.test.sdk</MicrosoftNetTestSdkPackageName>
```

Comment:
For public packages (like `Microsoft.Diagnostics.Tracing.TraceEvent`) we are using PascalCasing. Why use lowercasing for internal packages?

Reply:
It seems on the face of it to be a recipe for more of htese issues

Reply:
that's not related to internal vs public packages. We lower case the package name as the package folders are lowercased on Unix. Similar to https://github.com/dotnet/corefx/pull/33867/files/fe4aead92e8a08ba5fcde88dcd52ef1aeb2acdf1#diff-841e9e59d27acec708f2c73eaa3cf6a0R18 above. We use this property to access the folder in the filesystem below in a target.

Reply:
We probably also use that property for the package name but the property's intent is to map to the package folder name.

Reply:
ah ok